src/components/Dialog/DialogActions/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import useStyles from './styles';
+import {View, ViewProps} from 'react-native';
+
+interface IDialogActions extends ViewProps {}
+
+const DialogActions: React.FC<IDialogActions> = ({
+  children,
+  style,
+  ...rest
+}) => {
+  const styles = useStyles();
+  return (
+    <View style={[styles.actionsWrapper, style]} {...rest}>
```

Comment:
please, name the className the same as the component (dialogActions)

Reply:
React native doesn't need `className` as it uses styles not `classNames`, so I don't see any reason to use the same name for styles

Reply:
Just to be consistent with the way we name components and their styles in web and RN libraries